README.rst

```diff
@@ -118,12 +118,6 @@ Unit tests are provided within the source code. Integration, regression and syst
 Run pytest to run all unit tests (it will include the coverage).
 
 Acknowledgements
-~~~~~~~
+~~~~~~~~~~~~~~~~
 
 This work is co-funded by the EOSC-hub project (Horizon 2020) under Grant number 777536
-
-.. raw:: html 
-  <img src="https://wiki.eosc-hub.eu/download/attachments/1867786/eu%20logo.jpeg?version=1&modificationDate=1459256840098&api=v2" height="24"/>
-
-.. raw:: html 
-  <img src="https://wiki.eosc-hub.eu/download/attachments/18973612/eosc-hub-web.png?version=1&modificationDate=1516099993132&api=v2" height="24"/>
```

Comment:
Why removing the aboves?

Reply:
It seems this was added to workaround the fact that GitHub refuses to support scaling images in RST (https://github.com/github/markup/issues/295). Though if I look in the README on GitHub it doesn't even work and the images aren't displayed.

PyPI refuses to accept packages containing invalid RST or those containing disabled features (mostly raw HTML due to security concerns).

Other options:
* Use markdown instead of rst for the README
* Have a separate README for PyPI
* Resize the images manually and host them somewhere else

Reply:
Use markdown...?

Reply:
Sure, I'll convert it after lunch

Reply:
Actually GitHub has removed support for the syntax I was going to use so it's actually impossible to resize an image in a way which will work on both GitHub and PyPI.

Given that, I think our best solution would be to commit it somewhere under docs (it's only a few kB).

Reply:
..whatever works is fine...

Reply:
Done, assuming I renamed the repo/branch in the URL correctly it should now look like this once merged:

<img width="632" alt="Screenshot 2021-01-22 at 17 09 57" src="https://user-images.githubusercontent.com/5220533/105515360-e0d7e700-5cd4-11eb-85ef-0863f91c623a.png">